content/basics/ably.textile

```diff
@@ -0,0 +1,69 @@
+---
+title: What is Ably?
+meta_description: "Learn more about what Ably is and how it can power realtime digital experiences."
+---
+
+Ably is a platform that powers realtime digital experiences. Developers can build applications that communicate in realtime without needing to manage things such as scaling, latency, data integrity and network outages using Ably's SDKs and APIs.
+
+At its core Ably is a cloud-based pub/sub (publish/subscribe) platform-as-a-service (PaaS). It ensures that messages published to Ably by any device will be received, in realtime, by any number of subscribing devices. 
+
+Ably provides the realtime capabilities for applications across a wide range of industries, such as EdTech, FinTech and healthcare. The most common use cases Ably solves fall into the following categories:
```

Comment:
This list of industries is pretty specific and could be viewed as limiting. Maybe add SaaS (I know it's not an industry) and "sports and entertainment"?

Reply:
Good shout - I've updated to include those.